src/angular/styles/_typography.scss

```
@@ -809,6 +817,14 @@
       --sbb-panel-divider-color: var(--sbb-color-anthracite);
       --sbb-panel-border-color: var(--sbb-color-smoke);
 
+      // SBB Checkbox Panel & Radio Button Panel
+      --sbb-selection-panel-item-background-color: var(--sbb-color-iron);
+      --sbb-selection-panel-item-color: var(--sbb-color-graphite);
+      --sbb-selection-checked-border-color: var(--sbb-color-white);
```

```suggestion
      --sbb-selection-panel-border-color-checked: var(--sbb-color-white);
```

Please also change `--sbb-selection-disabled-border-color` to --sbb-selection-panel-border-color-disabled` here and in the other places.